include/aws/compression/compression.h

```diff
@@ -32,4 +34,6 @@ void aws_compression_library_init(struct aws_allocator *alloc);
 AWS_COMPRESSION_API
 void aws_compression_library_clean_up(void);
 
+AWS_POP_SANE_WARNING_LEVEL
```

Comment:
this file's lacking AWS_EXTERN_C_BEGIN/END

Reply:
Thanks, fixed.